make-prometheus-support.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo
+  echo "Usage:"
+  echo "  $ ./make-ops-viewer.sh [-r] <EMAIL_ADDRESS>"
```

Comment:
I think this could use an update; otherwise looks good!

Reply:
Good catch! Thank you! Fixed with next commit.